src/bugsnag.js

```diff
@@ -503,7 +503,7 @@
   // extract text content from a element
   function nodeText(el) {
     var text = el.textContent || el.innerText;
-    return truncate(text, 40);
+    return truncate(text.trim(), 40);
```

Comment:
Won't this break IE8 support? As `String.prototype.trim` was [first supported in IE9](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim#Browser_compatibility).